File: Chkeven.cpp
//template
#include<iostream>

using namespace std;


class Number
{
    private:
      int iNo;   //characterstics

      public:
      bool bflag=false;
       void Accept()  //setter
       {
           cout<<"enter the value\n";
           cin>>this->iNo;
       }

       void Display() // getter
       {
           cout<<"Value is:"<<this->iNo<<endl;
       }
       
       bool Chkeven()
       {
           if(iNo%2==0)
           {
               bflag=true;
               
           }
           return bflag;
       }
      
};

int main()
{
    Number nobj;
    bool bRet=false;

    nobj.Accept();
    nobj.Display();

    bRet=nobj.Chkeven();

    if(bRet==true)
    {
        cout<<"number is even\n";
    }
    else
    {
        cout<<"number is not even\n";
    }

    

    return 0;
}

File: Chkeven.java
import java.util.*;

class Number {
    private int iNo;
    public boolean bFlag;

    Number() {
        bFlag = false;
    }

    public void Accept() {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter Number:");
        this.iNo = sobj.nextInt();
    }

    public void Display() {
        System.out.println("Value is" + this.iNo);
    }

    public boolean Even() {
        if (iNo % 2 == 0) {
            bFlag = true;

        }

        return bFlag;
    }
}

public class Chkeven {
    public static void main(String arg[]) {
        Number nobj = new Number();
        boolean bRet = false;

        nobj.Accept();
        nobj.Display();

        bRet = nobj.Even();
        if (bRet == true) {
            System.out.println("number is even");

        } else {
            System.out.println("Not even");
        }

    }
}
